package.json

```diff
@@ -60,6 +60,7 @@
     "@storybook/preset-typescript": "^3.0.0",
     "@storybook/react": "~6.3.6",
     "@storybook/theming": "~6.3.6",
+    "@testing-library/jest-dom": "^5.14.1",
```

Comment:
Why you needed this one?

Reply:
This is for the `toBeDisabled` matcher.

See this issue: https://github.com/testing-library/dom-testing-library/issues/1003

Reply:
Why not `expect(input.disabled).toBe(true)`? It is the first time I see `input.matches`.

Reply:
`disabled` attribute is not set explicitly on the inputs, but on the fieldset